example/memcached-operator/memcached_controller.go.tmpl

```diff
@@ -1,141 +1,98 @@
-package memcached
+/*
```

Comment:
This file was just renamed from `example/kb-memcached-operator/memcached_controller.go.tmpl`